test/fixtures/api_namespaces.yml

```diff
@@ -109,6 +109,24 @@ monitoring_targets:
               }
   requires_authentication: false
   namespace_type: create-read-update-delete
+
+tls_monitoring_targets:
```

Comment:
can these dependents be moved to their own separate files? We want to prevent the model fixture file from getting overcrowded with plugin code